.github/workflows/test-starter-pack.yml

```diff
@@ -0,0 +1,64 @@
+name: Test starter pack
```

Comment:
Suggestion: Add a comment to indicate this is intended just for testing the starter pack itself. 

I believe a lot of teams clone the starter pack instead of using the init script, so this might help to prevent them from running these checks unnecessarily.

Reply:
Makes sense - but if teams keep cloning the starter pack in the future, they will be in a lot of trouble, so we should REALLY tell them not to do this.